src/pages/EditRequestPage.js

```diff
@@ -170,16 +155,6 @@ function EditRequestPage({report, route, policy, policyCategories, policyTags, p
         );
     }
 
-    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.CATEGORY && shouldShowCategories) {
```

Comment:
@dukenv0307 Make sure `shouldShowCategories` condition is migrated to IOURequestCategoryPage